pkg/reconciler/events/cloudevent/cloudevent.go

```diff
@@ -133,6 +133,9 @@ func EventForPipelineRun(pipelineRun *v1beta1.PipelineRun) (*cloudevents.Event,
 
 func getEventType(runObject objectWithCondition) (*TektonEventType, error) {
 	c := runObject.GetStatusCondition().GetCondition(apis.ConditionSucceeded)
+	if c == nil {
+		return nil, fmt.Errorf("no condition for in %T", runObject)
```

Comment:
nit: is the message missing a bit here? Looks like it should be `"no condition for %T"` or `"no condition for ConditionSucceeded in %T"` or similar?

Reply:
Nice catch, thank you!